sljit_src/sljitConfigInternal.h

```diff
@@ -580,10 +580,12 @@ typedef double sljit_f64;
 
 #endif /* !SLJIT_FPU_UNALIGNED */
 
-#if (defined SLJIT_CONFIG_X86_32 && SLJIT_CONFIG_X86_32)
-/* Auto detect SSE2 support using CPUID.
-   On 64 bit x86 cpus, sse2 must be present. */
-#define SLJIT_DETECT_SSE2 1
+#if (defined SLJIT_CONFIG_X86_32 && SLJIT_CONFIG_X86_32) || \
+	(defined SLJIT_CONFIG_ARM_V7 && SLJIT_CONFIG_ARM_V7)
+/* Auto detect availability of SSE2 (using CPUID) or NEON.
```

Comment:
It might be worth doing the checks here to avoid adding the "detect" support for cases that are known not to require it (ex: OpenBSD or macOS)